roles/reproducer/tasks/configure_controller.yml

```diff
@@ -314,26 +323,20 @@
         - bootstrap_layout
       when: cifmw_use_sushy_emulator | default(true) | bool
       block:
+        # Switch to import_role to ensure we load the parameters
+        # and environment, that are then consumed further down in
+        # generate_bm_info.yml
         - name: Deploy Sushy Emulator
-          vars:
-            cifmw_sushy_emulator_hypervisor_target: "{{ cifmw_target_host | default('localhost') }}"
-            cifmw_sushy_emulator_install_type: podman
-            cifmw_sushy_emulator_hypervisor_target_connection_ip: "{{ network_bridge_info['cifmw-public'] }}"
-          ansible.builtin.include_role:
+          ansible.builtin.import_role:
             name: sushy_emulator
 
-        - name: Let the project know we have Sushy Emulator available
-          ansible.builtin.set_fact:
-            _sushy_emulator_available: true
+        - name: Generate baremetal-info fact
```

Comment:
note that everything is now nested under the same `when` - though it might be an issue for vbmc use-case? @hjensas @lewisdenny lemme know.